app/router.js

```diff
@@ -7,6 +7,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('reminder');
```

Comment:
This is a route to the index of all reminders. So I think the route should be 'reminders'